README.md

```diff
@@ -1,2 +1,126 @@
+
+# Getting Started
+Below are the instructions on how to install, configure and generate a VMware vROps As Built report.
+
+## Pre-requisites
+The following PowerShell modules are required for generating a VMware vROps As Built report.
+
+Each of these modules can be easily downloaded and installed via the PowerShell Gallery 
+
+- [AsBuiltReport Module](https://www.powershellgallery.com/packages/AsBuiltReport/)
+- [VMware PowerCLI Module](https://www.powershellgallery.com/packages/VMware.PowerCLI/)
+- [PowervROPs Module](https://github.com/ymmit85/PowervROps/)
+
+### Module Installation
+
+Open a Windows PowerShell terminal window and install each of the required modules as follows;
+```powershell
+install-module AsBuiltReport
+install-module VMware.PowerCLI
+import-module PowervROPs
+```
+
+### Required Privileges
+
+To generate a VMware vROps report, a user account with Read-Only rights to all objects within vROps. Ideally a Local User Account is used.
+
+## Configuration
+The vROps As Built Report utilises a JSON file to allow configuration of report information, options, detail and healthchecks. 
+
+A vROps report configuration file can be generated by executing the following command;
+```powershell
+New-AsBuiltReportConfig -Report VMware.vROps -Path <User specified folder> -Name <Optional> 
+```
+
+Executing this command will copy the default vROps report JSON configuration to a user specified folder. 
+
+All report settings can then be configured via the JSON file.
+
+The following provides information of how to configure each schema within the report's JSON file.
+
+### Report
+The **Report** sub-schema provides configuration of the vSphere report information
+
+| Schema | Sub-Schema | Description |
+| ------ | ---------- | ----------- |
+| Report | Name | The name of the As Built Report
+| Report | Version | The report version
+| Report | Status | The report release status
+
+### Options
+The **Options** sub-schema allows certain options within the report to be toggled on or off
+
+| Schema | Sub-Schema | Setting | Description |
+| ------ | ---------- | ------- | ----------- |
+| Options | AuthSource | Local/Authentication Source Name | Set location of where to authenticate user running report.
+| Options | AlertFilter | String value | Used to report only on Alerts with specified value in name.
+
+
+### InfoLevel
+The **InfoLevel** sub-schema allows configuration of each section of the report at a granular level. The following sections can be set
+
+| Schema | Sub-Schema | Default Setting |
+| ------ | ---------- | --------------- |
+| InfoLevel | Authentication | 3
+| InfoLevel | Roles | 3
+| InfoLevel | Groups | 3
+| InfoLevel | Users | 3
+| InfoLevel | Adapters | 3
+| InfoLevel | RemoteCollectors | 3
+| InfoLevel | Alerts | 3
+| InfoLevel | Symptoms | 3
+| InfoLevel | SuperMetrics | 3
+| InfoLevel | ServiceStatus | 3
+| InfoLevel | CustomGroups | 3
+
+There are 6 levels (0-5) of detail granularity for each section as follows;
```

Comment:
Update as per previous comment and your intentions for the vROps report

Reply:
fixed